hack/hook-me.sh

```diff
@@ -256,122 +254,144 @@ keyUsage = nonRepudiation, digitalSignature, keyEncipherment
 extendedKeyUsage = clientAuth
 EOF
 
-  # Create a certificate authority
-  openssl genrsa -out ca.key 3072
-  openssl req -x509 -new -nodes -key ca.key -days 100000 -out ca.crt -subj "/CN=quic-tunnel-ca"
+    # Create a certificate authority
+    openssl genrsa -out ca.key 3072
+    openssl req -x509 -new -nodes -key ca.key -days 100000 -out ca.crt -subj "/CN=quic-tunnel-ca"
```

Comment:
```suggestion
    openssl req -x509 -new -nodes -key ca.key -days 1 -out ca.crt -subj "/CN=quic-tunnel-ca"
```
Please limit the validity to a day or so. This should be enough for debugging purposes.